src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4552,326 +4552,181 @@ internal static SessionStateAliasEntry[] BuiltInAliases
-                    new SessionStateAliasEntry("rm",
-                        "Remove-Item",     "", ScopedItemOptions.AllScope),
-                    new SessionStateAliasEntry("rmdir",
-                        "Remove-Item",     "", ScopedItemOptions.AllScope),
+                    new SessionStateAliasEntry("cat", "Get-Content"),
+                    new SessionStateAliasEntry("cp", "Copy-Item", "", AllScope),
+                    new SessionStateAliasEntry("ls", "Get-ChildItem"),
+                    new SessionStateAliasEntry("man", "help"),
+                    new SessionStateAliasEntry("mount", "New-PSDrive"),
+                    new SessionStateAliasEntry("mv", "Move-Item"),
+                    new SessionStateAliasEntry("ps", "Get-Process"),
+                    new SessionStateAliasEntry("rm", "Remove-Item"),
+                    new SessionStateAliasEntry("rmdir", "Remove-Item"),
 #endif
                     // Bash built-ins we purposefully keep even if they override native commands
-                    new SessionStateAliasEntry("cd",
-                        "Set-Location",    "", ScopedItemOptions.AllScope),
-                    new SessionStateAliasEntry("dir",
-                        "Get-ChildItem",   "", ScopedItemOptions.AllScope),
-                    new SessionStateAliasEntry("echo",
-                        "Write-Output",    "", ScopedItemOptions.AllScope),
-                    new SessionStateAliasEntry("fc",
-                        "Format-Custom",   "", ScopedItemOptions.ReadOnly | ScopedItemOptions.AllScope),
-                    new SessionStateAliasEntry("kill",
-                        "Stop-Process",    "", ScopedItemOptions.AllScope),
-                    new SessionStateAliasEntry("pwd",
-                        "Get-Location",    "", ScopedItemOptions.AllScope),
-                    new SessionStateAliasEntry("type",
-                        "Get-Content",     "", ScopedItemOptions.AllScope),
+                    new SessionStateAliasEntry("cd", "Set-Location", "", AllScope),
+                    new SessionStateAliasEntry("dir", "Get-ChildItem", "", AllScope),
+                    new SessionStateAliasEntry("echo", "Write-Output", "", AllScope),
+                    new SessionStateAliasEntry("fc", "Format-Custom", "", ReadOnly),
+                    new SessionStateAliasEntry("kill", "Stop-Process"),
+                    new SessionStateAliasEntry("pwd", "Get-Location"),
+                    new SessionStateAliasEntry("type", "Get-Content"),
                     // Native commands we keep because the functions act correctly on Linux
-                    new SessionStateAliasEntry("clear",
-                        "Clear-Host",      "", ScopedItemOptions.AllScope),
+                    new SessionStateAliasEntry("clear", "Clear-Host"),
 //#if !CORECLR is used to disable aliases for cmdlets which are not available on OneCore
 #if !CORECLR
```

Comment:
Should we just remove the !CORECLR section?

Reply:
I considered that - but maybe a couple still make sense - ise e.g. - and would `Out-Printer` come back?

Reply:
It's possible, so I guess it's best to leave it for now.